### ISSUE TITLE
Underscore issue

### ISSUE THREAD

**Issue by Carduelis** [erikvullings/strapi-to-typescript#34]
### Underscores inside relation fields are missing

I have a `members` collection:

```
// members
{
  "kind": "collectionType",
  "collectionName": "members",
  "attributes": {
    "users_permissions_user": {
      "plugin": "users-permissions",
      "model": "user",
      "via": "member"
    },
    "title": {
      "type": "string"
    },
    "supervised_projects": {
      "via": "supervisor",
      "collection": "projects"
    },
```

Which generates the following types:

```
export interface IMembers {
  id: string;
  userspermissions_user?: IUser;
  title?: string;
  supervisedprojects: IProjects[];
}
```

The noticeable part here is that only **the first underscore** is missing.

**Comment by Carduelis** [comment]
**UPD.**
I tried to find more information about the case and after reading the source code I realized where it comes from...

...drumroll...

...it comes from configuration example file!

```
  fieldName: (fieldName) => fieldName.replace("_", ""),
```

This thing is very dangerous, cause it may potentionally break the references.

My suggestion is:
1. Remove this thing from an example configuration file and add some comments that it can break stuff
2. Apply this function not only to a `fieldName` but also to a `fieldType` when it's a model type to keep references
